.overcommit.yml

```diff
@@ -29,6 +29,10 @@ PreCommit:
     command: ['yarn', 'run', 'standard']
     requires_files: true
     include: 'app/webpack/**/*.js'
+  Stylelint:
+    enabled: true
+    command: ['yarn', 'run', 'stylelint']
+    requires_files: true
```

Comment:
:wave: Just an observation from when I added the db anonymizer step below...
I believe that by setting requires_files you can set an include block that will make the task _only_ run if those files are changed.

With this setup, the step will run on **every** commit regardless of whether the S/CSS has been changed.

If you add 
```yaml
include: 'app/webpack/stylesheets/**/*.scss'
```
overcommit will only run when the scss files are changed and make it slightly faster :+1: